README.md

```diff
@@ -72,3 +72,12 @@ docker-compose up -d prometheus
 ```
 
 If the node is run by `node-private-slasher` or `node-public-slasher`, replace `node` in `depends_on` section of the prometheus service in the docker compose file to specify proper set of services to run.
+
+## Update repository to a newer version
+Update repository configs to a newer version should be as easy as pulling new changes from the repo and restarting the node.
+For example, the following sequence of commands should work in most of the cases. 
+```bash
+git pull
+docker-compose up down
```

Comment:
should it be `docker-compose down`? since the docker-compose file could be changed after `git pull` as so some service is removed does it make sense to stop the services before pulling the new version from git?

Reply:
Yes, that was a type.
For removing orphans, I suggest then to use `docker-compose down --remove-orphans`